src/test/java/tests/TestButtonTermsAndConditions.java

```diff
@@ -0,0 +1,18 @@
+package tests;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+import static java.lang.Thread.sleep;
+import static utills.CommonAction.scrollDown;
+
+public class TestButtonTermsAndConditions extends TestInit {
+    @Test(description = "SPAC-28")
+    public void checkButtonTermsAndConditions() throws InterruptedException {
+        scrollDown(driver);
```

Comment:
Why you used "scrollDown" and "sleep"?maybe it's not necessary?

Reply:
@DashaYab corrected